ci-operator/step-registry/openshift-qe/cluster-density-v2/openshift-qe-cluster-density-v2-commands.sh

```diff
@@ -23,6 +23,10 @@ current_worker_count=$(oc get nodes --no-headers -l node-role.kubernetes.io/work
 iteration_multiplier=$(($ITERATION_MULTIPLIER_ENV))
 export ITERATIONS=$(($iteration_multiplier*$current_worker_count))
 export WORKLOAD=cluster-density-v2
+echo "######"
+UUID="$(uuidgen)-cpt"
+export UUID
```

Comment:
Can we reduce this two into a single step? `export UUID="$(uuidgen)-cpt"`

Reply:
This was intentionally done to satisfy the shellcheck requirements.

Reply:
More info: https://www.shellcheck.net/wiki/SC2155

https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/43391/pull-ci-openshift-release-master-step-registry-shellcheck/1702614038594195456